whois-update/src/main/java/net/ripe/db/whois/update/authentication/strategy/OrgRefAuthentication.java

```diff
@@ -13,7 +13,6 @@
 import net.ripe.db.whois.update.domain.PreparedUpdate;
```

Comment:
I will change the name before merging to RefAuthentication.
I leave as OrgRefAuthentication for now for make it easier to review

Reply:
No please rename the class

Reply:
Done!